src/librustdoc/clean/mod.rs

```diff
@@ -272,7 +272,8 @@ impl Clean<Item> for doctree::Module<'_> {
 
 impl Clean<Attributes> for [ast::Attribute] {
     fn clean(&self, cx: &DocContext<'_>) -> Attributes {
-        Attributes::from_ast(cx.sess().diagnostic(), self, None)
+        let doc_cfg_active = cx.tcx.features().doc_cfg;
+        Attributes::from_ast(cx.sess().diagnostic(), self, None, doc_cfg_active, &cx.hidden_cfg)
```

Comment:
Why put this on `Attributes` instead of calling `tcx.features()` on demand?

Reply:
I found why - you use it directly in `from_ast`, not after `Attributes` is constructed. It's either this or pass a `tcx`; this seems fine.